extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/config/TracesBuildConfig.java

```diff
@@ -0,0 +1,102 @@
+package io.quarkus.opentelemetry.runtime.tracing.config;
+
+import static io.quarkus.opentelemetry.runtime.config.ExporterType.Constants.CDI_VALUE;
+import static io.quarkus.opentelemetry.runtime.tracing.config.TracesBuildConfig.SamplerType.Constants.*;
+
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.opentelemetry.runtime.config.ExporterType;
+import io.quarkus.opentelemetry.runtime.tracing.config.TracesBuildConfig.SamplerType.Constants;
+import io.smallrye.config.WithConverter;
+import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithParentName;
+
+public interface TracesBuildConfig {
+
+    /**
+     * Enable tracing with OpenTelemetry.
+     * <p>
+     * This property is not available in the Open Telemetry SDK. It's Quarkus specific.
+     * <p>
+     * Support for tracing will be enabled if OpenTelemetry support is enabled
+     * and either this value is true, or this value is unset.
+     */
+    @Deprecated()
+    @WithDefault("${quarkus.opentelemetry.tracer.enabled:true}")
+    Optional<Boolean> enabled();
+
+    /**
+     * List of exporters supported by Quarkus.
+     * <p>
+     * List of exporters to be used for tracing, separated by commas. none means no autoconfigured exporter.
+     * Has one of the values on {@link ExporterType} or the full qualified name of a class implementing
+     * {@link io.opentelemetry.sdk.trace.export.SpanExporter}
+     * <p>
+     * Default is OTLP.
+     *
+     * @return
+     */
+    @WithDefault(CDI_VALUE)
```

Comment:
This does not match the javadoc comment in my understanding

Reply:
will fix